net.go

```diff
@@ -1042,6 +1074,9 @@ func (m *Memberlist) sendLocalState(conn net.Conn, join bool, streamLabel string
 		}
 	}
 
+	moreBytes := binary.BigEndian.Uint32(bufConn.Bytes()[1:5])
+	metrics.SetGaugeWithLabels([]string{"memberlist", "size", "local"}, float32(moreBytes), m.metricLabels)
```

Comment:
suggestions for a better metric name?  (also, see question below about removing the remote metric, which might make naming this easier.)